skopt/learning/forest.py

```diff
@@ -3,6 +3,44 @@
 from sklearn.ensemble import ExtraTreesRegressor as sk_ExtraTreesRegressor
 
 
+def _return_std(X, trees, predictions):
+    """
+    Return standard deviation in predictions given a list of trees.
```

Comment:
Looks good, except for this docstring. We are computing `std(y|x)`, not `std(\hat(y)|x)`.